src/components/views/dialogs/ReportEventDialog.tsx

```diff
@@ -213,7 +214,7 @@ export default class ReportEventDialog extends React.Component<IProps, IState> {
         try {
             const client = MatrixClientPeg.get();
```

Comment:
Any end to end tests we can add to make sure pressing send report works?

Reply:
Possibly, though this code is in a bit of an awkward place to test. Would prefer to not be more invasive than I already am.